oldlog.go

```diff
@@ -97,6 +97,19 @@ func SetLogLevel(name, level string) error {
 	return nil
 }
 
+// GetSubsystems returns a slice containing the
+// names of the current loggers
+func GetSubsystems() []string {
+	subs := make([]string, len(loggers))
+
+	i := 0
+	for k := range loggers {
```

Comment:
And `append` needs to perform boundary checking every time. Anyway, I'm happy with the cleaner solution. Efficiency does not really weight in for 3-4 keys.